packages/components/src/alignment-matrix-control/types.ts

```diff
@@ -44,11 +44,32 @@ export type AlignmentMatrixControlIconProps = Pick<
 	AlignmentMatrixControlProps,
 	'value'
 > & {
+	/**
+	 * If `true`, disables pointer events on the icon.
+	 * @default true
+	 */
 	disablePointerEvents?: boolean;
+	/**
+	 * The width of the icon
+	 * @default 24
+	 */
+	width?: number;
+	/**
+	 * The height of the icon
+	 * @default 24
+	 */
+	height?: number;
+	/**
+	 * _Note: this prop is deprecated. Use the `width` and `height` props, or the
+	 * `size` prop on the parent `Icon` component instead_
+	 *
+	 * The size of the icon.
+	 * @ignore
+	 * @default 24
+	 */
```

Comment:
The `size` prop was simply more ergonomic than `width`/`height`, so it feels weird that we're highlighting the `width`/`height` props now. What do you think about _not_ adding explicit types and descriptions here? They will be added in "silently" through the `WordPressComponentProps<>`.

I am ok with soft deprecating the `size` prop — "just use the `size` prop on `Icon`" makes sense.

Reply:
Makes sense, even more so for the fact that `width` and `height` are legit SVG props. I applied your suggestions